sql/core/src/main/scala/org/apache/spark/sql/execution/WholeStageCodegenExec.scala

```diff
@@ -479,8 +482,7 @@ case class CollapseCodegenStages(conf: SQLConf) extends Rule[SparkPlan] {
   }
 }
 
-
-case class WholeStageCodegenRDD(@transient sc: SparkContext, var source: CodeAndComment,
+private[spark] class WholeStageCodegenRDD(@transient sc: SparkContext, var source: CodeAndComment,
```

Comment:
Any specific reason for changing the it to a non-case class? This class was specifically made a case class to improve concurrency. See https://github.com/SnappyDataInc/spark/pull/63